processes/procstat.go

```diff
@@ -76,18 +76,18 @@ func (psc *procStatsCollector) GetStats(procPath string) (map[string][]Proc, err
 			fstat := filepath.Join(procPath, file.Name(), procStat)
 			procStat, err := ioutil.ReadFile(fstat)
 			if err != nil {
-				return nil, err
+				continue
```

Comment:
@katarzyna-z again this agree that this is a good practice in general but in this case I am really wondering if it is worth implementing because we are sure that this happens when
there is a de-sync between when the list of processes has been collected and the time taken to open the appropriate entry in the filesystem therefore the error as the process is gone.

Furthermore, looking at the code of other collectors, there are not so many calls to log.ANYTHING() (or logger.ANYTHING()) and the one that I have tried to implement when
I debugged some of these plugins never showed up in snapd logs whatever the level I used
both on the plugin and daemon side

Please advise